knowledge-base/adr/outdated/0010-test-strategy.md

```diff
@@ -1,3 +1,5 @@
+status **superseded** The turborepo takes care of recompilation and the content is not really a ADR.
```

Comment:
[pp] I would delete instead of move. It's git... you can find deleted stuff if you want.

Reply:
in principle you are right, but if the file is not present anymore you have to dig deep. I would rather keep it for the moment.

Reply:
Alternatively I would vote to keep it in the same folder but have a sort of overview markdown which lists all active ADRs.

Reply:
then one has to maintain the overview markdown. I ask the colleagues in the chat.

Reply:
ne clear preference - I like the folder so I would go for it